src/app/cli/src/client.ml

```diff
@@ -720,6 +720,50 @@ let set_staking =
                (Public_key.Compressed.to_base58_check
                   (Public_key.compress public_key)) ))
 
+(* A step towards `account import`, for now `unsafe-import` will suffice *)
+let unsafe_import =
+  Command.async
+    ~summary:
+      "Unsafely import a password protected private key to one with the \
+       password stripped, but tracked by this daemon and accessible via the \
+       GraphQL API"
+    (let open Command.Let_syntax in
+    (* We'll do this entirely without talking to the daemon for now, though in the future this may change *)
+    let%map_open privkey_path = Cli_lib.Flag.privkey_read_path
+    and conf_dir = Cli_lib.Flag.conf_dir in
+    fun () ->
+      let open Deferred.Let_syntax in
+      let%bind home = Sys.home_directory () in
+      let conf_dir =
+        Option.value ~default:(home ^/ Cli_lib.Default.conf_dir_name) conf_dir
+      in
+      let wallets_disk_location = conf_dir ^/ "wallets" in
+      let%bind ({Keypair.public_key; _} as keypair) =
+        Secrets.Keypair.Terminal_stdin.read_exn privkey_path
+      in
+      let pk = Public_key.compress public_key in
+      let%bind wallets =
+        Secrets.Wallets.load ~logger:(Logger.create ())
+          ~disk_location:wallets_disk_location
+      in
+      (* Either we already are tracking it *)
+      match Secrets.Wallets.find wallets ~needle:pk with
+      | Some _ ->
+          printf
+            !"Key already present, no need to import : %s\n"
+            (Public_key.Compressed.to_base58_check
```

Comment:
`(Public_key.Compressed.to_base58_check(Public_key.compress public_key))` is used like 50 times in our codebase lol. We should create a method `Public_key.to_compressed_base58_check`

Reply:
I think it's out-of-scope of this PR to handle that, but I created an issue so someone can do that later https://github.com/CodaProtocol/coda/issues/3150